quill-cassandra/src/main/scala/io/getquill/source/cassandra/CassandraSource.scala

```diff
@@ -0,0 +1,163 @@
+package io.getquill.source.cassandra
+
+import scala.language.higherKinds
+import scala.language.experimental.macros
+import scala.util.Try
+
+import com.datastax.driver.core.Session
+
+import io.getquill._
+import io.getquill.quotation.Quoted
+import io.getquill.naming.NamingStrategy
+import io.getquill.source.Source
+
+trait CassandraSource[N <: NamingStrategy, R, S]
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [Number of methods in class exceeds 30](https://www.codacy.com/app/fwbrasil/quill/file/1759497974/issues/source?bid=2746728&fileBranchId=3046581#l14)